src/sap.m/src/sap/m/MessageToast.js

```diff
@@ -374,9 +446,32 @@ sap.ui.define(['jquery.sap.global', './InstanceManager', 'sap/ui/core/Popup'],
 			// validate all settings
 			this._validateSettings(mSettings);
 
+			// if content is a string, set as a text control for backward compatibility
+			if (typeof oContent === "string") {
+				oContent = new sap.m.Text({
+					text: oContent,
+					textAlign: "Center"
+				});
+			}
+
+			// group content into a single container control so the popup instance key can be added later
+			oContentContainer = new sap.m.FlexBox({
```

Comment:
> would that be jQuery.sap.require?

It depends... for modules that are always needed or "usually anyway loaded" it would be another entry at the very top of the file in the sap.ui.define arguments list.
For heavyweight stuff that is usually not needed and also not typically used by other modules, it would rather be a lazy require. Could be done synchronously with jQuery.sap.require (but we are getting rid of all sync stuff) or sap.ui.require if possibly async.

But for the moment I'd suggest to first get general feedback before tweaking single issues.

Regards
Andreas